.travis.yml

```diff
@@ -31,9 +31,23 @@ matrix:
       - pip install tox
       script: tox -e django18
       env:
-        name: Django 1.8 Unit Tests
+        name: Python 27 and Django 1.8 Unit Tests
     - install:
       - pip install tox
       script: tox -e django111
       env:
-        name: Django 1.11 Unit Tests
+        name: Python 27 and Django 1.11 Unit Tests
+    - install:
+      - pip install tox
+      python:
+        - "3.5"
+      script: tox -e django18
+      env:
+        name: Python 35 and Django 1.8 Unit Tests
+    - install:
+      - pip install tox
+      python:
+        - "3.5"
+      script: tox -e django111
+      env:
+        name: Python 35 and Django 1.11 Unit Tests
```

Comment:
@noisecapella Could you please review the change of this travis file. It looks like build is stuck.

Initially I have tried `tox -e py35-django18` but it was failing on travis saying `ERROR: InterpreterNotFound: python3.5`. After updating file its working now but stuck on travis, not sure if there is an issue with the configuration.

CC: @pdpinch